### ISSUE TITLE
Add new static Answers RETURNS_DEFAULT_VALUE

### ISSUE THREAD

**Issue by rymsha** [mockito/mockito#2104]
- [ ] The mockito message in the stacktrace have useful information, but it didn't help
 - [ ] The problematic code (if that's possible) is copied here;
       Note that some configuration are impossible to mock via Mockito
 - [ ] Provide versions (mockito / jdk / os / any other relevant information)
 - [ ] Provide a [Short, Self Contained, Correct (Compilable), Example](http://sscce.org) of the issue
       (same as any question on stackoverflow.com)
 - [ ] Read the [contributing guide](https://github.com/mockito/mockito/blob/release/3.x/.github/CONTRIBUTING.md)

Mockito v3.6.0

Add new static `Answer` to `Answers`: `Answers.RETURNS_DEFAULT_VALUE` that calls method's `getDefaultValue`
This would be useful for annotation mocking, especially with `MockitoExtension` `@Mock(answer = Answers.RETURNS_DEFAULT_VALUE)`

Background:
In OSGi Annotations are used as "Component Property Type":
```
public @interface MyConfig { String timeout() default "1s";}

...

@Activate
public void activate(MyConfig config) {timeout = config.timeout();}
```
verifying the method `activate` and configuration property default values is problematic.

Current workaround:
`mock(MyConfig.class, invocation -> invocation.getMethod().getDefaultValue());`

**Comment by TimvdLippe** [comment]
I think this is a duplicate of #1900

**Comment by rymsha** [comment]
Yeah, trying to solve the same problem with a different approach, basically.
